changelog.txt

```diff
@@ -546,7 +547,7 @@ Added
 Changed
 +++++++
 
- - The MasterCrawler logic has been simplified; their primary function is the compilation of index documents from slave crawlers, all export logic, including data mirroring is now provided by the ``signac.export()`` function.
+ - The MasterCrawler logic has been simplified; their primary function is the compilation of index documents from subcrawlers, all export logic, including data mirroring is now provided by the ``signac.export()`` function.
```

Comment:
```suggestion
 - The MainCrawler logic has been simplified; their primary function is the compilation of index documents from subcrawlers, all export logic, including data mirroring is now provided by the ``signac.export()`` function.
```
Not sure if this is confusing, but might make sense to also make the substitution here.

Reply:
@csadorf I was unsure about that -- at first, I left references in the changelog for previous versions because I wasn't sure if that "rewrote history" -- but on second thought, that's the intention of this PR. I have updated the changelog accordingly.